tests/gem5/gpu/test_gpu_pannotia.py

```diff
@@ -0,0 +1,344 @@
+        "-c",
+        joinpath(binary_path, "color_maxmin.gem5"),
+        "--options",
+        f'{joinpath(dataset_path, "ecology1.graph")} 0',
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-color-maxmin-g3-circuit",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "color_maxmin.gem5"),
+        "--options",
+        f'{joinpath(dataset_path, "G3_circuit.graph")} 0',
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-color-max-ecology",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "color_max.gem5"),
+        "--options",
+        f'{joinpath(dataset_path, "ecology1.graph")} 0',
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-color-max-g3-circuit",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "color_max.gem5"),
+        "--options",
+        f'{joinpath(dataset_path, "G3_circuit.graph")} 0',
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-fw-hip-256",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "fw_hip.gem5"),
+        "--options",
+        f'-f {joinpath(dataset_path, "256_16384.gr")} -m default',
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-fw-hip-512",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "fw_hip.gem5"),
+        "--options",
+        f'-f {joinpath(dataset_path, "512_65536.gr")} -m default',
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+# This test fails with
```

Comment:
I have a fix for this locally but was not aware it was failing for people in public -- I thought it was only for the multi-chiplet support (not on mainline).

Same idea for MIS though about input size: https://github.com/gem5/gem5/blob/stable/tests/deprecated/weekly.sh#L324